packages/bridge/src/ContractAdapter.ts

```diff
@@ -1,9 +1,18 @@
-import { Logger } from './Logger';
-import { EntityConfig } from './type-definition/EntityConfig';
+import type { Logger } from './Logger';
+import type { EntityConfig } from './type-definition/EntityConfig';
 
 export interface ContractAdapter {
   contractName: string;
   entityConfigs: EntityConfig[];
   logger?: Logger;
+
+  /**
+   * With this option enabled, the generated smart-contract will not store any data to state-data
+   * https://github.com/lecle/aloxide/issues/49
+   *
+   * Default is false
+   */
+  logDataOnly?: boolean;
```

Comment:
Boolean variable isn't necessary optional if we have default value for it e.g. `false` here.

Reply:
The optional setting here means we don't have to have that property in a Typescript object.